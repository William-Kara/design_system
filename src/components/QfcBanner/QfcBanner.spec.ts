import { it, describe, beforeEach, expect } from 'vitest';

import { shallowMount } from '@vue/test-utils';
import QfcQfcBanner from './QfcBanner.vue';

let wrapper: any = null;

const propsValues = {};

beforeEach(() => {
  wrapper = shallowMount(QfcQfcBanner, {
    propsData: propsValues,
  });
});

describe('QfcBanner Component', () => {
  it('should mount properly', () => {
    expect(wrapper).toBeTruthy();
  });

  it('should have correct props', () => {
    expect(wrapper.props()).toEqual(propsValues);
  });

  it('should render correctly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
