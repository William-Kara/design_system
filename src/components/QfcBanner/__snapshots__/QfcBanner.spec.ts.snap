// Vitest Snapshot v1

exports[`QfcBanner Component > should render correctly 1`] = `
"<section data-v-a99819f7=\\"\\" class=\\"qfc-banner hero\\">
  <div data-v-a99819f7=\\"\\" class=\\"qfc-banner__body hero-body\\" style=\\"background-image: url(undefined);\\"></div>
</section>"
`;
